aws/efs_test.go

```diff
@@ -0,0 +1,158 @@
+package aws
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/efs"
+	"github.com/aws/aws-sdk-go/aws"
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/go-commons/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type TestElasticFileSystem struct {
+	Name *string
+	ID   *string
+}
+
+func createTestElasticFileSystem(t *testing.T, session *session.Session, name string) (*TestElasticFileSystem, error) {
```

Comment:
Returned `error` is `nil` each time, maybe should be handled `err` from `cfg, err := awsconfig.LoadDefaultConfig`

Reply:
Thanks, Denis! Fixed in [efffae3](https://github.com/gruntwork-io/cloud-nuke/pull/353/commits/efffae353f0ba70afe35b337380bbc931ca467ba)